Makefile

```diff
@@ -269,9 +275,10 @@ endif
 ifeq ($(origin DEV_BANK_PRIVKEY),undefined)
 	echo "<DEV_BANK_PRIVKEY> environment variable missing" >&2 && exit 1
 endif
+	$(GET_PEER_ID)
```

Comment:
What goes `$(GET_PEER_ID)` do and where is it defined?

Reply:
It should be `${PEER_ID}` It was renamed but not fully checked.

Reply:
Renamed